shell/platform/android/io/flutter/embedding/engine/mutatorsstack/FlutterMutatorsStack.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.embedding.engine.mutatorsstack;
+
+import android.graphics.Matrix;
+import android.graphics.Path;
+import android.graphics.Rect;
+import android.graphics.RectF;
+import androidx.annotation.Keep;
+import androidx.annotation.NonNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Keep
+public class FlutterMutatorsStack {
+
+  private @NonNull List<FlutterMutator> mutators;
```

Comment:
`FlutterMutator` could be an embedded class.

Reply:
done